cluster_utils/settings.py

```diff
@@ -222,6 +248,16 @@ def sanitize_numpy_torch(possibly_np_or_tensor):
 
 
 def save_metrics_params(metrics, params):
+    """Send results to the cluster_utils server.
+
+    Call this function at the end of your job script to report the results back to
+    cluster_utils.
+
+    Args:
+        metrics:  Dictionary with metrics that should be sent to the server.
```

Comment:
Add type of dictionary, it should be Dict[str, float]

(the function also accepts numpy arrays/torch tensors and tries to convert them to float, but the intended inputs are only floats)